app/serializers/admin_order_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class AdminOrderSerializer
```

Comment:
This serializer will send back all the necessary data needed for the front end that deals with orders